app/views/licences/tabs/returns.njk

```diff
@@ -21,7 +21,7 @@
 
 <table class="govuk-table">
   <h2 class="govuk-heading-l">Returns</h2>
-  {% if returns.length > 0 %}
+  {% if hasReturns %}
     <thead class="govuk-table__head">
```

Comment:
thought from the other pr. 

See if this can use the njk component. 

Will mark as draft whilst working on this.

Reply:
<img src="https://media1.giphy.com/media/xUA7aP05OSTwjR51Ic/giphy.gif"/>